test/test_zx.cpp

```diff
@@ -409,3 +413,45 @@ TEST_F(ZXTest, TwoQubitRotations) {
   EXPECT_EQ(ecm->getResults().equivalence,
             ec::EquivalenceCriterion::Equivalent);
 }
+
+TEST_F(ZXTest, EmptyCircuitWithAncillas) {
+  auto qc1 = qc::QuantumComputation(0U);
+  qc1.addAncillaryRegister(1);
+  qc1.x(0);
+
+  auto qc2 = qc::QuantumComputation(0U);
+  qc2.addAncillaryRegister(1);
+
+  config.execution.runZXChecker = true;
+  config.execution.parallel = false;
+  config.execution.runSimulationChecker = false;
+  config.execution.runAlternatingChecker = false;
+  config.execution.runConstructionChecker = false;
+  ecm = std::make_unique<ec::EquivalenceCheckingManager>(qc1, qc2, config);
+
+  ecm->run();
+  EXPECT_EQ(ecm->getResults().equivalence,
+            ec::EquivalenceCriterion::NoInformation);
```

Comment:
I think it would make sense to run a second check with the new option and assert that it yields the expected result (equivalent).

Reply:
Good idea, done.